api/src/db/DynamoBusinessesDataClient.test.ts

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import { DynamoDBDocumentClient } from "@aws-sdk/lib-dynamodb";
+import { dynamoDbTranslateConfig } from "@db/config/dynamoDbConfig";
+import { DynamoBusinessesDataClient } from "@db/DynamoBusinessesDataClient";
+import { BusinessesDataClient } from "@domain/types";
+import { generateBusiness, generateProfileData, generateTaxFilingData } from "@shared/test";
+import dayjs from "dayjs";
+
+// references jest-dynalite-config values
+const dbConfig = {
+  tableName: "businesses-table-test",
+};
+
+describe("DynamoBusinessesDataClient", () => {
+  const config = {
+    endpoint: process.env.MOCK_DYNAMODB_ENDPOINT,
+    sslEnabled: false,
+    region: "local",
+  };
+
+  let client: DynamoDBDocumentClient;
+  let dynamoBusinessesDataClient: BusinessesDataClient;
+
+  beforeEach(() => {
+    client = DynamoDBDocumentClient.from(new DynamoDBClient(config), dynamoDbTranslateConfig);
+    dynamoBusinessesDataClient = DynamoBusinessesDataClient(client, dbConfig.tableName);
+  });
+
+  it("Test to verify db gets inserted items", async () => {
```

Comment:
[pebble] not sure if this was left from the initial draft of the test but it doesn't read well, "it Test to....". You could rewrite it as `it("successfully inserts and retrieves items from the db")`